src/runtime/dom-particle.ts

```diff
@@ -153,14 +162,14 @@ export class DomParticle extends XenStateMixin(DomParticleBase) {
     this._setProps(props);
   }
 
-  async _addNamedHandleData(dictionary, handleName) {
+  private async _addNamedHandleData(dictionary, handleName) {
     const handle = this.handles.get(handleName);
     if (handle) {
       dictionary[handleName] = await this._getHandleData(handle);
     }
   }
 
-  async _getHandleData(handle: Handle) {
+  private static async _getHandleData(handle: Handle) {
```

Comment:
Mixed opinions on moving helpers like this to static.  Lint will want you to  use the fully qualified class name instead of `this.` which is often less readable.

Reply:
Agreed -- changed in 8a35d1b